pkg/solana/txm/txm.go

```diff
@@ -568,6 +583,65 @@ func (txm *Txm) Enqueue(accountID string, tx *solanaGo.Transaction, txCfgs ...Se
 	return nil
 }
 
+func (txm *Txm) EstimateAndSetComputeUnitLimit(ctx context.Context, tx *solanaGo.Transaction) (err error) {
```

Comment:
can we share logic with the existing simulate call here?

Reply:
Good point. I've updated the code to remove some duplicity here